packages/go_router/test/go_router_test.dart

```diff
@@ -1650,6 +1650,11 @@ class DummyBuildContext implements BuildContext {
     throw UnimplementedError();
   }
 
+  @override
```

Comment:
Couldn't you temporarily comment this `@override` out and ignore the lint warning, with a TODO to restore it later, so that it would work on stable?

Reply:
Yes, I definitely need to do that. I only ran the analyzer for the exact version of master where this was introduced.

Reply:
Done